lib/vsc/utils/fancylogger.py

```diff
@@ -85,6 +85,15 @@
 import weakref
 from distutils.version import LooseVersion
 
+HAVE_COLOREDLOGS_MODULE = False
+if os.environ.get('FANCYLOGGER_NO_COLOREDLOGS', '0').lower() not in ('1', 'yes', 'true', 'y'):
```

Comment:
well, it's not like it's a lot of work? I wouldn't mind if that's included in this PR, since the changes done here are 'expoising' the issue...